src/pybind/mgr/mgr_module.py

```diff
@@ -126,6 +126,34 @@ def get_crush_version(self):
     def dump(self):
         return self._dump()
 
+    def get_pool(self, pool_name):
+        # FIXME: efficient implementation
+        d = self._dump()
+        for p in d['pools']:
+            if p['pool_name'] == pool_name:
+                return p
+
+        return None
+
+    def get_pool_by_id(self, pool_id):
+        # FIXME: efficient implementation
+        d = self._dump()
+        for p in d['pools']:
+            if p['pool'] == pool_id:
+                return p
+
+        return None
```

Comment:
Same here. raise `KeyError`?

Reply:
turns out we dn't use these accessors, just dropping them!